core/src/main/scala/com/spotify/featran/transformers/HashOneHotEncoder.scala

```diff
@@ -17,27 +17,51 @@
 
 package com.spotify.featran.transformers
 
+import scala.math.ceil
```

Comment:
this should be right above `scala.util` based on default Intellij import format.